docs/route/action.md

```diff
@@ -65,4 +65,5 @@ See also:
 
 [form]: ../components/form
 [form action]: ../components/form#action
-[index query param]: ../guides/routing#what-is-the-index-query-param
+<!-- page doesn't exist anymore -->
+[index query param]: ../discussion/02-routes#what-is-the-index-query-param
```

Comment:
This section (https://remix.run/docs/en/dev/discussion/02-routes#what-is-the-index-query-param) doesn't seem to exist anymore

Reply:
Same as above